pkg/targets/adapter/splunktarget/adapter_test.go

```diff
@@ -18,14 +18,15 @@ package splunktarget
 
 import (
 	"context"
-	"net/http"
+	"fmt"
 	"testing"
 	"time"
 
-	"github.com/stretchr/testify/assert"
-
 	cloudevents "github.com/cloudevents/sdk-go/v2"
+	"github.com/stretchr/testify/assert"
 
+	"github.com/triggermesh/triggermesh/pkg/apis/targets/v1alpha1"
+	targetce "github.com/triggermesh/triggermesh/pkg/targets/adapter/cloudevents"
```

Comment:
Care to separate these two from knative.dev imports?

Reply:
got it!